models/demos/wormhole/mistral7b/demo/demo_with_prefill.py

```diff
@@ -338,40 +390,47 @@ def run_mistral_demo(user_input, batch_size, device, instruct_mode, is_ci_env, n
                         logger.trace(f"[User {user}] Finished decoding at iteration {iteration}")
                         if all(user_done):
                             users_decoding = False
+            profiler.end(f"log_printing_iter_{iteration}", iteration=batch_idx)
 
+            profiler.start(f"decode_embedding", iteration=batch_idx)
             if embed_on_device:
                 tt_out_tok = ttnn.from_torch(tt_out_tok, device=device, dtype=ttnn.uint32, layout=ttnn.ROW_MAJOR_LAYOUT)
                 pt_encoded_input = tt_embd(tt_out_tok)
             else:
                 pt_encoded_input = embd(tt_out_tok)
+            profiler.end(f"decode_embedding", iteration=batch_idx)
 
             # Print out generated outputs for each user at the end of every iteration
             iteration_time = time() - iteration_time_start
             tokens_per_second_per_user = 1 / iteration_time
             # Print out generated outputs for each user at the end of every iteration
-            if not is_ci_env:
-                if len(user_input) == 1:
-                    logger.info("[User 0] {}".format("".join(tokenizer.decode(all_outputs[0]))))
-                else:
-                    for user in range(batch_size):
-                        text = "".join(tokenizer.decode(all_outputs[user]))
-                        if len(text) > 100:
-                            text = "..." + text[-97:]
-                        text = text.replace("\n", " ")
-                        logger.info("[User {}] {}".format(user, text))
+            # if not is_ci_env:
```

Comment:
Should this be uncommented again?

Reply:
I'll uncomment for the time being. 

The reason to leave this comment out in both Mistral and Llama is due to batching already printing the output at the end. I fear it might be to much output, but we can always change this back later on.